packages/docs/docs/advanced-customization/custom-templates.md

```diff
@@ -76,7 +76,7 @@ render(
 );
 ```
 
-You also can provide your own field template to a uiSchema by specifying a `ui:ArrayFieldTemplate` property.
+You also can provide your own field template to a uiSchema by specifying a `ui:ArrayFieldTemplate` property with your Component or a string value from the `Registry`.
```

Comment:
Consider adding at least one example of how to use an alternate template with a different name in the `uiSchema` here and possibly all the sections you've updated below